src/components/Card.js

```diff
@@ -1,51 +1,97 @@
 import React from 'react';
 
+import DownloadIcon from '@material-ui/icons/CloudDownload';
+import { ThemeProvider } from 'styled-components/macro';
 import { makeStyles } from '@material-ui/core/styles';
 import Paper from '@material-ui/core/Paper';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
 
 import { TextLoader, ChartLoader } from './Loader';
-import BarChart from './BarChart';
+import { ButtonStyle } from './Button';
+import Chart from './Chart';
 
 import PropTypes from 'prop-types';
 
+const theme = {
+  fg: 'white',
+  bg: '#468DAE',
+  border: '1px',
+  width: '150px',
+  padding: '1px',
+  shadow:
+    '0px 1px 3px 0px rgba(0,0,0,0.2), 0px 1px 1px 0px rgba(0,0,0,0.14), 0px 2px 1px -1px rgba(0,0,0,0.12)'
+};
+
 const cardStyle = makeStyles(theme => ({
   root: {
     paddingBottom: theme.spacing(2)
   },
-  cardProperty: {
-    padding: theme.spacing(1, 1)
+  cardWrapper: {
+    padding: theme.spacing(2, 2)
+  },
+  iconProperty: {
+    paddingRight: '5px'
   }
 }));
 
 const Card = ({ content }) => {
   const classes = cardStyle();
 
   const loaderType = content.type === 'text' ? <TextLoader /> : <ChartLoader />;
+  const subItems =
+    content.type === 'pieChart' ? (
+      <Box>
+        <Typography m={1}>Data sharing: {content.dataPolicy}</Typography>
+        <Typography m={1}>Sample units: {content.sampleUnits} </Typography>
+      </Box>
+    ) : null;
+
+  const subTitle = content.subTitle ? (
+    <Typography variant="h6">{content.subTitle}</Typography>
+  ) : null;
+
+  const downLoadButton =
```

Comment:
Instead of using a ternary that returns null, you can also use the following pattern:

```js
const showIfTrue = booleanThing && <h1>It's true</h1>
```